cam.go

```diff
@@ -150,21 +158,36 @@ func init() {
 			fc.acceptedStats.startTime = time.Now()
 			fc.rejectedStats.startTime = time.Now()
 
+			// Initialize the image buffer
+			imageFreq := newConf.ImageFrequency
+			if imageFreq == 0 {
+				imageFreq = defaultImageFreq
+			}
+			fc.buf = imagebuffer.NewImageBuffer(newConf.WindowSeconds, imageFreq)
+
+			// Initialize background image capture worker
+			fc.backgroundWorkers = utils.NewStoppableWorkerWithTicker(
```

Comment:
This background worker polls and stores the camera at a set frequency regardless of how long it takes the trigger evaluation to happens